tests/h/search/index_test.py

```diff
@@ -48,72 +46,6 @@ def test_it_indexes_presented_annotation(
         )
         assert indexed_ann == presenter.asdict.return_value
 
-    def test_it_can_index_an_annotation_with_no_document(
-        self, factories, index_annotations, get_indexed_ann
-    ):
-        annotation = factories.Annotation.build(document=None)
-
-        index_annotations(annotation)
-
-        assert get_indexed_ann(annotation.id)["document"] == {}
```

Comment:
Yeah this seems to just be testing a default behaviour of Elasticsearch? Would it be possible for it _not_ to be able to index an annotation with no document?

I guess this test doesn't do much harm and might protect us against a future Elasticsearch version that disallows indexing `{}` for the document, but it seems unlikely that a future Elasticsearch version would change in that way, and this mostly seems to be a test for `DocumentSearchIndexPresenter` (which is what renders `annotation.document` as `{}` if it's `None`)